jsonrpc/dispatcher.go

```diff
@@ -434,6 +435,16 @@ func (d *Dispatcher) getBlockHeaderImpl(number BlockNumber) (*types.Header, erro
 	}
 }
 
```

Comment:
I would go even a step further and suggest we add a comment for this method that explains what it does. What does `available` mean in this context? I think it would be clearer if there was an explanation

Reply:
This method is removed